### PR TITLE
Reference darwin.builder and historical context

### PR DESCRIPTION
**Pull request by puffnfresh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+In 2018, QEMU got experimental support for Hypervisor.framework and
+that got promoted to stable in 2019. QEMU is now fast and since
+nixpkgs has great support for building and running QEMU virtual
+machines, there's little need for this project.
+
+---
+
 # LinuxKit Nix - Linux on Mac Nix builder
 
 [hydra jobset](https://hydra.nixos.org/jobset/linux-on-mac-builder/master)
```
Comment (zowoq) [inline README.md:35]: Can probably disable this jobset?

Comment (zimbatm) [reply README.md:35]: Good idea, I disabled it

Comment (zowoq) [reply README.md:35]: Thanks!

**Comment by zimbatm** [comment]
Maybe this repo show be archived entirely. @nicknovitski?

**Comment by zowoq** [comment]
It's been a few weeks now so I'll archive the repo.

**Pull request merged**